contracts/duckies/RandomUpgradeable.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+contract RandomUpgradeable is Initializable {
```

Comment:
I don't understand what is this file for

Reply:
This is a peculiarity of Upgradeable contracts. As Random also holds storage variables and Garden inherits from it, Random should be declared Upgradeable and inherit from Initializable.